src/test/core/Config_test.cpp

```diff
@@ -767,9 +647,9 @@ trustthesevalidators.gov
     {
         detail::RippledCfgGuard const cfg(*this, "testSetup",
             explicitPath ? "test_db" : "", "");
-        /* ConfigGuard has a Config object that gets loaded on construction,
-            but Config::setup is not reentrant, so we need a fresh config
-            for every test case, so ignore it.
+        /* RippleCfgGuard has a Config object that gets loaded on
```

Comment:
should be `RippledCfgGuard `

Reply:
Ugh. These typos are going to be the death of me.